src/AppInstallerRepositoryCore/RepositorySource.cpp

```diff
@@ -43,8 +43,8 @@ namespace AppInstaller::Repository
             bool IsTombstone = false;
         };
 
-        // Finds a source from the given vector by its name.
-        auto FindSourceByName(std::vector<SourceDetailsInternal>& sources, std::string_view name)
+        // Finds a source from the given vector by its name. Returned source cannot be modified.
+        auto FindSourceByNameConst(const std::vector<SourceDetailsInternal>& sources, std::string_view name)

```

Comment:
>FindSourceByNameConst [](start = 13, length = 21)

Does not appear to be used (and if it was the comment would be to move it to SourceListInternal).

Reply:
oops, forgot to clean up this one.

---
In reply to: [520185301](https://github.com/microsoft/winget-cli/pull/634#discussion_r520185301) [](ancestors = 520185301)